src/code/graph.c

```diff
@@ -159,7 +159,9 @@ void Graph_Destroy(GraphicsContext* gfxCtx) {
 }
 
 void Graph_TaskSet00(GraphicsContext* gfxCtx) {
+#if OOT_DEBUG
     static Gfx* sPrevTaskWorkBuffer = NULL;
+#endif
```

Comment:
does this have to be in-function to match? I would prefer it outside, just to get the ifdef clutter out of the function personally

Reply:
This variable if of utmost interest for debugging gfx crashes so if it could be out of function and without `static` (so it appears in the map) it would be fantastic

Reply:
(this conversation can happen elsewhere probably) but, is it? When using tharos gbd for example i usually only need to grab the address of `gGfxPools`. I guess I am uneducated as to what the work buffer even is

Reply:
on second thought ive never actually used gbd myself, tharo has helped me every time. so hes probably manually calculating the location of the desired buffer based on the location of the gfxpools :D

Reply:
Unfortunately I'm not having much luck moving it outside the function, it changes codegen

Reply:
welp nevermind then lmao